.github/workflows/codespell.yml

```diff
@@ -0,0 +1,20 @@
+---
+name: Codespell
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  codespell:
+    name: Check for spelling errors
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: codespell-project/actions-codespell@master
+        with:
+          skip: js,v*.rst,_stop_words.py
+          ignore_words_file: .github/codespell_ignore_words.txt
```

Comment:
Can this be placed in a RC file so it is easier to run locally? (https://github.com/codespell-project/codespell#using-a-config-file)

Reply:
Good idea, in Python projects I can put these in `setup.cfg`.